package.json

```diff
@@ -2,12 +2,12 @@
   "name": "standard-version",
   "version": "3.0.0",
   "description": "replacement for `npm version` with automatic CHANGELOG generation",
-  "bin": "cli.js",
+  "bin": "bin/standard-version.js",
```

Comment:
the naming needs to be changed. `cli` should be the `bin`. So you'd need to rename the files to whatever you think makes more sense :)